test/common/unit.js

```diff
@@ -273,4 +273,11 @@ describe('Unit', () => {
       command: 'select * from myTable where id in (@param1_0, @param1_1, @param1_2)'
     })
   })
+
+  it('tagged template literal request', () => {
```

Comment:
Can you add a test explicitly testing the callback behaviour, please?

Reply:
There is no callback in template() per say, it's just to enable the use of the callback version of query().